CHANGELOG.md

```diff
@@ -52,6 +52,9 @@ amount, based on time since the last failure. The credit is now: `2 ^ (minutes s
 ## Component Updates and Bug Fixes
 
 * github.com/openziti/ziti: [v1.4.3 -> v1.5.0](https://github.com/openziti/ziti/compare/v1.4.3...v1.5.0)
+    * [Issue #2835](https://github.com/openziti/ziti/issues/2835) - Add mechanism for selecting CLI layout
```

Comment:
This linked ticket has no information about what this is, what it does, or why it exists.

Either the issue needs to be expanded upon or some release notes need to be added.

Reply:
Updated the issue.